src/molecules/formfields/CreditCardField/credit_card_field.module.scss

```diff
@@ -1,45 +0,0 @@
-$form_border: 1px solid color('neutral-5');
```

Comment:
maybe we should keep this individual scss file instead of dumping everything in `shared`?